boards/arm/beagle_bcf/beagleconnect_freedom_defconfig

```diff
@@ -25,3 +25,6 @@ CONFIG_HW_STACK_PROTECTION=y
 CONFIG_SERIAL=y
 CONFIG_CONSOLE=y
 CONFIG_UART_CONSOLE=y
+
+# Adjust for oscillator capacitors
+CONFIG_CC13X2_CC26X2_XOSC_CAPARRAY_DELTA=0x02
```

Comment:
Your commit message does not mention about touching this file/ please explain in commit message.